docs/templates/metrics.md

```diff
@@ -19,7 +19,7 @@ model.compile(loss='mean_squared_error',
 
 A metric function is similar to a [loss function](/losses), except that the results from evaluating a metric are not used when training the model.
 
-You can either pass the name of an existing metric, or pass a Theano/TensorFlow symbolic function (see [Custom metrics](#custom-metrics)).
+You can either pass the name of an existing metric, or pass a function or layer (see [Custom metrics](#custom-metrics)).
```

Comment:
You can either pass the name of an existing metric, a metric function, or a layer (see...)

Reply:
"pass a layer" is not sound advice in the general case. 

For now I would not be comfortable covering usage of stateful metrics in the public docs. This is an API that is still experimental, and subject to changes in the future (e.g. this PR changes it). We'll document it when it is more mature.

Please revert the edits on this page. You can keep them around and we may include them at a future date.

Reply:
Makes sense! Will do.